modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/util/ExportImportLayoutHelperImpl.java

```diff
@@ -774,11 +752,31 @@ public MissingReferences validateImportPortletInfo(
 		}
 	}
 
+	private static final Snapshot<ImportController>
```

Comment:
Hey @GabrielHenriquedev, why do you need a `Snapshot` here? Please explain this in the commit message.